assets/model_monitoring/components/src/data_quality_compute_metrics/compute_data_quality_metrics.py

```diff
@@ -2,8 +2,7 @@
 # Licensed under the MIT License.
 
```

Comment:
Update the component version and all the graph that consume this component

Reply:
Done